commons/src/main/java/info/metadude/android/eventfahrplan/commons/Clock.kt

```diff
@@ -0,0 +1,42 @@
+package info.metadude.android.eventfahrplan.commons
```

Comment:
Move requested classes to temporal package, as well as network.utils.DateParser to network.temporal.DateParser

Reply:
Please apply the package change to the initial commit where the class was introduced so that we do not have to deal with the move in the history. Git should handle this pretty easily. Just use a interactive rebase to edit the initial commit.

Reply:
Done with latest version of this branch, see e02cd22d1d9ed64bb90671f32fcf52386b4b42bc.